docs/001_develop/03_client-capabilities/008_interaction/section-navigator.mdx

```diff
@@ -0,0 +1,569 @@
+  detail: { sectionName: 'section1', stateName: 'active' },
+  bubbles: true
+});
+navigator.dispatchEvent(event);
+
+// Method 2: Without sectionName (auto-detected from slot)
+// Button is inside the section slot
+<rapid-section-navigator>
+  <div slot="section1">
+    <h2>Section 1 Content</h2>
+    <button onclick="handleSetActive(event)">
+      Set Active State
+    </button>
+  </div>
+</rapid-section-navigator>
+
+function handleSetActive(event) {
+  const button = event.target;
+  const setStateEvent = new CustomEvent('section-navigator-set-state', {
+    detail: { stateName: 'active' },
+    bubbles: true
+  });
+  button.dispatchEvent(setStateEvent);
+  // Component auto-detects 'section1' from slot context
+}
+```
+</td>
+    </tr>
+    <tr>
+      <td>section-navigator-set-states</td>
+      <td><code>{'{'} sectionName?: string, stateNames: string[] {'}'}</code></td>
+      <td>Listens for events to add multiple states to a section. If `sectionName` is omitted and event is dispatched from within a slot, the component auto-detects the section from the slot context.</td>
+<td>
+
+```typescript
+// Method 1: With explicit sectionName (dispatch on component)
+const navigator = document.querySelector('rapid-section-navigator');
+const event = new CustomEvent('section-navigator-set-states', {
+  detail: { sectionName: 'section1', stateNames: ['active', 'pending'] },
+  bubbles: true
+});
+navigator.dispatchEvent(event);
+
+// Method 2: Without sectionName (auto-detected from slot)
+<rapid-section-navigator>
+  <div slot="section1">
+    <h2>Section 1 Content</h2>
+    <button onclick="handleSetMultipleStates(event)">
+      Add Multiple States
+    </button>
+  </div>
+</rapid-section-navigator>
+
+function handleSetMultipleStates(event) {
+  const button = event.target;
+  const setStatesEvent = new CustomEvent('section-navigator-set-states', {
+    detail: { stateNames: ['active', 'pending'] },
+    bubbles: true
+  });
+  button.dispatchEvent(setStatesEvent);
+  // Component auto-detects 'section1' from slot context
+}
+```
+</td>
+    </tr>
+    <tr>
+      <td>section-navigator-clear-state</td>
+      <td><code>{'{'} sectionName?: string, stateName: string {'}'}</code></td>
+      <td>Listens for events to remove a specific state from a section. If `sectionName` is omitted and event is dispatched from within a slot, the component auto-detects the section from the slot context.</td>
+<td>
+
+```typescript
+// Method 1: With explicit sectionName (dispatch on component)
+const navigator = document.querySelector('rapid-section-navigator');
+const event = new CustomEvent('section-navigator-clear-state', {
+  detail: { sectionName: 'section1', stateName: 'active' },
+  bubbles: true
+});
+navigator.dispatchEvent(event);
+
+// Method 2: Without sectionName (auto-detected from slot)
+<rapid-section-navigator>
+  <div slot="section1">
+    <h2>Section 1 Content</h2>
+    <button onclick="handleClearState(event)">
```

Comment:
These examples are good, but for now you should change the event bindings to be web components rather than angular to match other files